### PR TITLE
Fix protoc script generator to use @grpc/grpc-js

### LINKED ISSUE #3934: [grpc] generate typings use `grpc` instead of `grpc/grpc-js`
### Bug description

It is a regression. We fixed it already once. 😕 

### Steps to reproduce

NA

### Expected behavior

_No response_

### Example repository

_No response_

### PR DESCRIPTION
**Pull request by aledbf**
Migrate from package `grpc` to `@grpc/grpc-js` in the code generation script and apply all the required changes in the codebase.

Note: the removal of dependencies in yarn is due to a change in `grpc_tools_node_protoc_ts@5.1.3` that removed the  `handlebars-helpers` dependency

fixes #3934

### CODE REVIEW

File: components/ws-manager-bridge/src/bridge-controller.ts [hunk 1]
```diff
@@ -86,9 +86,11 @@ export class BridgeController {
         const bridge = this.bridgeFactory() as WorkspaceManagerBridge;
         const clientProvider = async () => {
             const grpcOptions = {
-                "grpc.keepalive_timeout_ms": 1500,
-                "grpc.keepalive_time_ms": 1000,
+                "grpc.keepalive_timeout_ms": 20000,
+                "grpc.http2.min_time_between_pings_ms": 1000,
```
Comment (aledbf) [inline components/ws-manager-bridge/src/bridge-controller.ts:90]: These two settings are too aggresive. Without changing the values I get the next error
```console
{"@type":"type.googleapis.com/google.devtools.clouderrorreporting.v1beta1.ReportedErrorEvent","serviceContext":{"service":"ws-manager-bridge","version":"aledbf-grpcjs.19"},"stack_trace":"Error: 8 RESOURCE_EXHAUSTED: Bandwidth exhausted\n    at Object.callErrorFromStatus (/app/node_modules/@grpc/grpc-js/build/src/call.js:31:26)\n    at Object.onReceiveStatus (/app/node_modules/@grpc/grpc-js/build/src/client.js:327:49)\n    at Object.onReceiveStatus (/app/node_modules/@grpc/grpc-js/build/src/client-interceptors.js:299:181)\n    at /app/node_modules/@grpc/grpc-js/build/src/call-stream.js:145:78\n    at processTicksAndRejections (internal/process/task_queues.js:79:11)","component":"ws-manager-bridge","severity":"ERROR","time":"2021-04-14T23:53:04.248Z","environment":"devstaging","region":"europe-west1","message":"wsman subscription error","error":"Error: 8 RESOURCE_EXHAUSTED: Bandwidth exhausted\n    at Object.callErrorFromStatus (/app/node_modules/@grpc/grpc-js/build/src/call.js:31:26)\n    at Object.onReceiveStatus (/app/node_modules/@grpc/grpc-js/build/src/client.js:327:49)\n    at Object.onReceiveStatus (/app/node_modules/@grpc/grpc-js/build/src/client-interceptors.js:299:181)\n    at /app/node_modules/@grpc/grpc-js/build/src/call-stream.js:145:78\n    at processTicksAndRejections (internal/process/task_queues.js:79:11)","payload":{"name":"dev","url":"ws-manager:8080"}}
``` 

Is there a particular reason for those values? @csweichel @akosyakov

**Comment by aledbf** [comment]
After the migration and update of the code and connection values, this is the output of the log:
```console
k logs -f ws-manager-bridge-765546f7ff-tp54f
yarn run v1.22.4
warning Skipping preferred cache folder "/.cache/yarn" because it is not writable.
warning Selected the next writable cache folder in the list, will be "/tmp/.yarn-cache-31001".
$ node ./dist/ee/src/index.js
warning Cannot find a suitable global folder. Tried these: "/usr/local, /.yarn"
{"component":"ws-manager-bridge","severity":"INFO","time":"2021-04-15T00:03:18.929Z","environment":"devstaging","region":"europe-west1","message":"Loading TypeORM entities and migrations from /app/node_modules/@gitpod/gitpod-db/lib/typeorm","loggedViaConsole":true}
{"component":"ws-manager-bridge","severity":"INFO","time":"2021-04-15T00:03:18.931Z","environment":"devstaging","region":"europe-west1","message":"Using DB: db:3306/gitpod"}
{"component":"ws-manager-bridge","severity":"INFO","time":"2021-04-15T00:03:19.069Z","environment":"devstaging","region":"europe-west1","message":"Trying to connect to RabbitMQ","payload":{"URL":"amqps://override-me:****@messagebus:5672"}}
{"component":"ws-manager-bridge","severity":"DEBUG","time":"2021-04-15T00:03:19.069Z","environment":"devstaging","region":"europe-west1","message":"Try 1..."}
{"component":"ws-manager-bridge","severity":"INFO","time":"2021-04-15T00:03:19.093Z","environment":"devstaging","region":"europe-west1","message":"Connection to RabbitMQ established"}
{"component":"ws-manager-bridge","severity":"INFO","time":"2021-04-15T00:03:19.112Z","environment":"devstaging","region":"europe-west1","message":"Initializing Jaeger Tracer with RemoteReporter and ConstSampler(always)","loggedViaConsole":true}
{"component":"ws-manager-bridge","severity":"INFO","time":"2021-04-15T00:03:19.133Z","environment":"devstaging","region":"europe-west1","message":"prometheus metrics server running on: 9500"}
{"component":"ws-manager-bridge","severity":"INFO","time":"2021-04-15T00:03:19.137Z","environment":"devstaging","region":"europe-west1","message":"reconcile: create bridge for new cluster","payload":{"name":"dev"}}
{"component":"ws-manager-bridge","severity":"INFO","time":"2021-04-15T00:03:19.147Z","environment":"devstaging","region":"europe-west1","message":"Initializing Jaeger Tracer with RemoteReporter and ConstSampler(always)","loggedViaConsole":true}
{"component":"ws-manager-bridge","severity":"INFO","time":"2021-04-15T00:03:19.157Z","environment":"devstaging","region":"europe-west1","message":"Initializing Jaeger Tracer with RemoteReporter and ConstSampler(always)","loggedViaConsole":true}
{"component":"ws-manager-bridge","severity":"INFO","time":"2021-04-15T00:03:19.159Z","environment":"devstaging","region":"europe-west1","message":"starting bridge to cluster...","payload":{"name":"dev","url":"ws-manager:8080"}}
{"component":"ws-manager-bridge","severity":"DEBUG","time":"2021-04-15T00:03:19.159Z","environment":"devstaging","region":"europe-west1","message":"starting DB updater: dev","payload":{"name":"dev","url":"ws-manager:8080"}}
{"component":"ws-manager-bridge","severity":"INFO","time":"2021-04-15T00:03:19.160Z","environment":"devstaging","region":"europe-west1","message":"attempting to establish wsman subscription","payload":{"name":"dev","url":"ws-manager:8080"}}
{"component":"ws-manager-bridge","severity":"DEBUG","time":"2021-04-15T00:03:19.161Z","environment":"devstaging","region":"europe-west1","message":"starting controller: dev","payload":{"name":"dev","url":"ws-manager:8080"}}
{"component":"ws-manager-bridge","severity":"INFO","time":"2021-04-15T00:03:19.161Z","environment":"devstaging","region":"europe-west1","message":"started bridge to cluster.","payload":{"name":"dev","url":"ws-manager:8080"}}
{"component":"ws-manager-bridge","severity":"DEBUG","time":"2021-04-15T00:03:19.163Z","environment":"devstaging","region":"europe-west1","message":"using TLS config to connect ws-manager"}
{"component":"ws-manager-bridge","severity":"INFO","time":"2021-04-15T00:03:19.175Z","environment":"devstaging","region":"europe-west1","message":"ws-manager-bridge is up and running"}
{"component":"ws-manager-bridge","severity":"INFO","time":"2021-04-15T00:03:19.178Z","environment":"devstaging","region":"europe-west1","message":"gRPC server listening on: localhost:8080"}
{"component":"ws-manager-bridge","severity":"DEBUG","time":"2021-04-15T00:04:19.162Z","environment":"devstaging","region":"europe-west1","message":"controlling instances","payload":{"installation":"dev"}}
{"component":"ws-manager-bridge","severity":"INFO","time":"2021-04-15T00:04:19.182Z","environment":"devstaging","region":"europe-west1","message":"reconcile: cluster already present, doing nothing","payload":{"name":"dev"}}
{"component":"ws-manager-bridge","severity":"DEBUG","time":"2021-04-15T00:05:19.161Z","environment":"devstaging","region":"europe-west1","message":"controlling instances","payload":{"installation":"dev"}}
{"component":"ws-manager-bridge","severity":"INFO","time":"2021-04-15T00:05:19.185Z","environment":"devstaging","region":"europe-west1","message":"reconcile: cluster already present, doing nothing","payload":{"name":"dev"}}
{"@type":"type.googleapis.com/google.devtools.clouderrorreporting.v1beta1.ReportedErrorEvent","serviceContext":{"service":"ws-manager-bridge","version":"aledbf-grpcjs.21"},"component":"ws-manager-bridge","severity":"ERROR","time":"2021-04-15T00:05:30.445Z","environment":"devstaging","region":"europe-west1","message":"error sending spans over HTTP: Error: connect ETIMEDOUT 10.63.254.35:14268","loggedViaConsole":true}
{"@type":"type.googleapis.com/google.devtools.clouderrorreporting.v1beta1.ReportedErrorEvent","serviceContext":{"service":"ws-manager-bridge","version":"aledbf-grpcjs.21"},"component":"ws-manager-bridge","severity":"ERROR","time":"2021-04-15T00:05:30.445Z","environment":"devstaging","region":"europe-west1","message":"Failed to flush spans in reporter: error sending spans over HTTP: Error: connect ETIMEDOUT 10.63.254.35:14268","loggedViaConsole":true}
{"component":"ws-manager-bridge","severity":"DEBUG","time":"2021-04-15T00:06:19.162Z","environment":"devstaging","region":"europe-west1","message":"controlling instances","payload":{"installation":"dev"}}
{"component":"ws-manager-bridge","severity":"INFO","time":"2021-04-15T00:06:19.188Z","environment":"devstaging","region":"europe-west1","message":"reconcile: cluster already present, doing nothing","payload":{"name":"dev"}}
{"component":"ws-manager-bridge","severity":"DEBUG","time":"2021-04-15T00:07:19.161Z","environment":"devstaging","region":"europe-west1","message":"controlling instances","payload":{"installation":"dev"}}
{"component":"ws-manager-bridge","severity":"INFO","time":"2021-04-15T00:07:19.191Z","environment":"devstaging","region":"europe-west1","message":"reconcile: cluster already present, doing nothing","payload":{"name":"dev"}}
{"component":"ws-manager-bridge","severity":"DEBUG","time":"2021-04-15T00:08:19.161Z","environment":"devstaging","region":"europe-west1","message":"controlling instances","payload":{"installation":"dev"}}
{"component":"ws-manager-bridge","severity":"INFO","time":"2021-04-15T00:08:19.194Z","environment":"devstaging","region":"europe-west1","message":"reconcile: cluster already present, doing nothing","payload":{"name":"dev"}}
{"component":"ws-manager-bridge","severity":"DEBUG","time":"2021-04-15T00:09:19.162Z","environment":"devstaging","region":"europe-west1","message":"controlling instances","payload":{"installation":"dev"}}
{"component":"ws-manager-bridge","severity":"INFO","time":"2021-04-15T00:09:19.197Z","environment":"devstaging","region":"europe-west1","message":"reconcile: cluster already present, doing nothing","payload":{"name":"dev"}}
```

**Review by akosyakov — APPROVED** [review]
I was able to compile Gitpod Code again against this PR. But better if @csweichel looks at internal usage changes.

File: components/ws-manager-bridge/src/bridge-controller.ts (hunk 1 above)
Comment (csweichel) [reply components/ws-manager-bridge/src/bridge-controller.ts:90]: We originally choose those in a time where bridge would open too many connections and swamp ws-manager. I reckon choosing more conservative values should be fine, especially if we keep tabs on ws-manager's memory consumption.

**Review by csweichel — APPROVED** [review]
I don't think I understand all possible consequences here. But conceptually the change makes sense to me.

**Pull request merged**